distribution/docs/src/main/resources/content/_integrating/_endpoints/query-endpoints.adoc

```diff
@@ -0,0 +1,241 @@
+:title: Queries Endpoint
+:type: endpoint
+:status: published
+:operations: crud
+:link: _queries_endpoint
+:summary: To perform CRUD (Create, Read, Update, Delete) operations on query metacards in the catalog, work with one of these endpoints.
+
+== {title}
+
+The queries endpoint enables an application to create, retrieve, update, and delete query metacards.
+
+The queries endpoint can be used for one or more of these operations on an instance of ${branding}:
+
+* Create query metacards and store them in the ${branding} catalog.
+* Retrieve all query metacards stored in the ${branding} catalog and sort them based on attribute and sort order.
+* Retrieve a specific query metacard stored in the ${branding} catalog.
+* Update query metacards that are stored in the ${branding} catalog.
+* Delete query metacards that are stored in the ${branding} catalog.
+
+.Queries Endpoint URL
+[source,http]
+----
+https://<HOSTNAME>:<PORT>/search/catalog/internal/queries
+----
+
+=== Queries Endpoint Create Examples
+
+To create a query metacard through the queries endpoint, send a `POST` request to the queries endpoint.
+
+.Queries Endpoint Create Request Body
+[source,json]
```

Comment:
can you make this `[source,json,linenums]`? It's nice to have the line numbers on samples longer than two or three lines